cv32e40s/fv/Makefile

```diff
@@ -35,12 +35,21 @@ DPI_DASM_SPIKE_REPO ?= TODO_dontwanttodefinethis
 include  $(CORE_V_VERIF)/mk/fv/fv.mk
 
 
-# Make target
+# Safety Check
 
 ifndef  CV_SIM_PREFIX
   $(warning  CV_SIM_PREFIX undefined)
 endif
 
-fv:  $(CV_CORE_PKG)
+
+# Make Targets
+
+## JasperGold
+jg:  $(CV_CORE_PKG)
 	$(CV_SIM_PREFIX)  jaspergold  jg.tcl
 	# TODO:robin-silabs  Move to "mk/fv/jg.mk"?
+
```

Comment:
`q` is a really short target name.  Maybe `questa_formal`?

Reply:
What about... using the same name as the executable?

That would be `jaspergold.tcl` and `qverify.tcl`.

The make targets could have "longhand" names correspondingly:
```
jaspergold: jg
jg: <etcetc>
  <etcetc>

qverify: q
q: <etcetc>
  <etcetc>
```

Reply:
Works for me.